corehq/apps/app_manager/static/app_manager/json/template_apps/appcues.json

```diff
@@ -0,0 +1,1191 @@
+{
```

Comment:
Out of curiosity, how did you generate this?

Reply:
https://www.commcarehq.org/a/template-apps/apps/download/ae67c8f583012d45d1270e458a93036c/ - Amelia & Sheel built this app ages ago when we first released a template apps feature.